docs/embeddings/roboflow-api.md

```diff
@@ -0,0 +1,38 @@
+---
+---
+
+# Roboflow
+
+You can use [Roboflow Inference](https://inference.roboflow.com) with Chroma to calculate multi-modal text and image embeddings with CLIP. through the `RoboflowEmbeddingFunction` class. Inference can be used through the Roboflow cloud, or run on your hardware.
+
+## Roboflow Cloud Inference
+
+To run Inference through the Roboflow cloud, you will need an API key. [Learn how to retrieve a Roboflow API key](https://docs.roboflow.com/api-reference/authentication#retrieve-an-api-key). 
+
+You can pass it directly on creation of the `RoboflowEmbeddingFunction`:
+
+```python
+from chromadb.utils.embedding_functions import RoboflowEmbeddingFunction
+
+roboflow_ef = RoboflowEmbeddingFunction(api_key=API_KEY)
+```
+
+Or else you can set it as an environment variable:
+
+```bash
+export ROBOFLOW_API_KEY=YOUR_API_KEY
```

Comment:
See also https://github.com/chroma-core/chroma/pull/1434/files#r1429551932 - it would be good if the EF picked up the env. variable automatically if it exists.